fusionbackend/src/main/java/io/fusion/fusionbackend/dto/mappers/AssetDetailsMapper.java

```diff
@@ -21,23 +21,65 @@
 import io.fusion.fusionbackend.model.AssetType;
 import io.fusion.fusionbackend.model.Company;
 import io.fusion.fusionbackend.model.Room;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 
 import java.util.Set;
 
 @Component
 public class AssetDetailsMapper extends EntityDetailsDtoMapper<Asset, AssetDetailsDto> {
+    private final BaseAssetMapper baseAssetMapper;
```

Comment:
Find ich gut, dass der BaseAssetMapper benutzt wird. Ein AssetDetailsDto ist ja ein Subtyp von BaseAssetDto. Sollte man das dann nicht auch mit dem AssetMapper machen. AssetDto ist auch in der Hierarchie.